docs/admin-api.md

```diff
@@ -0,0 +1,204 @@
+# Admin API
+
+
+## Token
+
+The token class contains the majority of the logic for communicating with shopify.
+To use the token class, you must define a child class for the tokens you are using
+and implement a subset of the `save` and `load` abstract methods. Your option for
+child classes are `OnlineTokenABC`, `OfflineTokenABC` or `PrivateTokenABC`:
+
+### Implement Token Classes
+
+```python
+class OfflineToken(OfflineTokenABC):
+  async def save(self):
+      # Some code to save the token to a database
+
+  @classmethod
+  async def load(cls, store_name: str):
+      # Some code to load the token from the database
+
+class OnlineToken(OnlineTokenABC):
+  async def save(self):
+      # Some code to save the token to a database
+
+  @classmethod
+  async def load(cls, store_name: str, user_id: str):
+      # Some code to load the token from the database
+
+class PrivateToken(PrivateTokenABC):
+  @classmethod
+  async def load(cls, store_name: str, user_id: str):
```

Comment:
just saw this, probably don't need `user_id` for private app token?

Reply:
Please make an issue to review this properly since it's not the point of this PR.

Reply:
though it does look like just a documentation error since the code doesn't have the `user_id` param
https://github.com/SatelCreative/spylib/blob/main/spylib/token.py#L311

Reply:
We could have separate PR to fix documentation error if any since this PR is strictly reorganizing the content?